### DISCUSSION TITLE
Failed running examples from installation section of wiki

### DISCUSSION THREAD

**Discussion by Neel-Shah-29** [boostorg/ublas#173] [Q&A] [answered]
I am getting the following error when i tried to run the examples in installation section of wiki. 

![image](https://user-images.githubusercontent.com/84740927/155088353-6f0de484-d8ef-43b6-b66f-2ebd6b384a5f.png)

 Also i have the file allocater_access.hpp in the same directory.

![image](https://user-images.githubusercontent.com/84740927/155089114-39343b27-4639-412a-b7b9-e671722fcae2.png)

What is the possible solutions to this?

Note: I have build ublas and run tests successfully.

User-config.jam
```
using gcc : : g++ : <cxxflags>-std=c++20 <linkflags> <link>-lboost_unit_test_framework ;
```
Possible fixes i tried:
1) Changing the flags in user-config.jam
2) Changing the include path of allocater_access.hpp

**Comment by amitsingh19975** [comment, marked as answer]
*Fix:
```jam
using gcc : : g++ : <cxxflags>"-std=c++20" <linkflags>"-lboost_unit_test_framework" ;
```
To know more about the flags, go and read the `Boost.Build` documentation, which can be found [here](https://www.boost.org/build/).

If you don't want to use the configuration file, you can command-line arguments to run the test.

```bash
PATH_TO_THE_B2_BINARY/b2 toolset=gcc cxxstd=20 variant=release optimization=speed PATH_TO_THE_TEST_OR_EXAMPLES_FOLDER

PATH_TO_THE_B2_BINARY/b2 toolset=gcc cxxstd=20 PATH_TO_THE_TEST_OR_EXAMPLES_FOLDER

PATH_TO_THE_B2_BINARY/b2 toolset=gcc cxxstd=20 variant=release PATH_TO_THE_TEST_OR_EXAMPLES_FOLDER
```
If you get the link issue from the boost unit test, use the linking flag.

**Reply by Neel-Shah-29** [reply]
yes i will try to use command line arguments to solve the issue. Thanks for the suggestion.

**Reply by Neel-Shah-29** [reply]
Using Command line arguments solves the problem.I am attaching the screenshots of output for future reference if needed.

![image](https://user-images.githubusercontent.com/84740927/155402988-7fb6ac2b-3065-41e2-86cc-4e3f334d0201.png)

![image](https://user-images.githubusercontent.com/84740927/155403253-ab688205-85c8-4ad7-bd65-9bcbad1b3921.png)

**Comment by amitsingh19975** [comment]
The `allocator_access.hpp` should be the part of standard boost library if you have installed the boost from the package manager or used GitHub. It does not relate to the `Boost.uBLAS`, but if you are having an issue, I recommend you get a copy of the header file and paste it into the appropriate path.

**Reply by Neel-Shah-29** [reply]
ok sure will do it
